server/routerlicious/packages/routerlicious-base/src/alfred/services/documentDeleteService.ts

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IDocumentDeleteService } from "@fluidframework/server-services-core";
+
+export class DocumentDeleteService implements IDocumentDeleteService {
+	constructor() {}
+
+	public async deleteDocument(tenantId: string, documentId: string): Promise<void> {
```

Comment:
>Promise<void> [](http://example.com/codeflow?start=68&length=13)

Let's check FRS delete logic to see if we'd like to return something?

Reply:
On FRS side, return type of deleteContainer method is Promise<void>. The container delete doesn't return anything for successful cases. For failure case, it throws an ArmError (this will need some refactor if request is not through RPASS in future, but this would be FRS side implementation details then) with proper status code and error message.